jest.config.js

```diff
@@ -1,12 +1,7 @@
 module.exports = {
-  collectCoverageFrom: [
-    '<rootDir>/app/scripts/**/*.(js|ts|tsx)',
-    '<rootDir>/shared/**/*.(js|ts|tsx)',
-    '<rootDir>/ui/**/*.(js|ts|tsx)',
-  ],
   coverageDirectory: './coverage',
   coveragePathIgnorePatterns: ['.stories.*', '.snap'],
-  coverageReporters: ['html', 'json'],
+  coverageReporters: ['json'],
```

Comment:
HTML reports are the sole method I use to review coverage. I use them at least. The JSON report isn't human readable, we can't really use that to review coverage.

Reply:
If we're not uploading it anywhere in CI, we can disable the HTML reporter there though.

Reply:
@Gudahtt  If I see it correctly, they are not uploaded or used anywhere in CI, which is why I removed it. But I am not adamant about this at all, we can put it back if you think it is useful to always generate html reports. For your manual coverage reviews, you can still use the html flag to override the config defaults.

Reply:
What I mean is that this should be the default when locally running unit tests. The HTML report is the only way to review coverage locally. If we don't have that, it's hardly worth running tests with coverage at all.

Reply:
Oh, I understand now. Thank you for the clarification. I propose the following to keep both the performance and development experience benefits:

```ts
process.env.CI ? ['json'] : ['html', 'json'],
```

Is this okay with you? 😄

Reply:
Yep, makes sense!

Reply:
resolving as fixed, thank you for the suggestion!

Reply:
:+1: